x/emissions/module/autocli.go

```diff
@@ -572,6 +572,15 @@ func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
 						{ProtoField: "block_height_last_inference"},
 					},
 				},
+				{
+					RpcMethod: "GetNetworkInferencesAtBlockOutlierResistant",
+					Use:       "network-inferences-at-block-outlier-resistant [topic_id] [block_height_last_inference]",
+					Short:     "Get the Network Inferences for a topic at a block height where the last inference was made removing outliers",
+					PositionalArgs: []*autocliv1.PositionalArgDescriptor{
+						{ProtoField: "topic_id"},
+						{ProtoField: "block_height_last_inference"},
```

Comment:
Should have a query to obtain this if we don't already.

And/or consider making a `latest-network-inference-outlier-resistant` query where the block arg is not even needed, paralleling the existing network inference query.

Reply:
true, will replicate the latest too.

Reply:
Done for 
network-inferences-at-block-outlier-resistant
latest-network-inferences-outlier-resistant
latest-available-network-inferences-outlier-resistant